packages/tradershub/src/hooks/useTotalAssets.ts

```diff
@@ -0,0 +1,46 @@
+import { useActiveTradingAccount, useAuthorize } from '@deriv/api';
+import { Helpers } from '@deriv/library';
+import { useUIContext } from '../components';
+import useCFDAssets from './useCFDAssets';
+import usePlatformAssets from './usePlatformAssets';
+
+/**
+ * @description Get total balance of cfd and platform accounts
+ * @returns data - Total balance of all cfd and platform accounts
+ */
+const useTotalAssets = () => {
+    const { getUIState } = useUIContext();
```

Comment:
@thisyahlen-deriv suppose you have to change this because this [PR](https://github.com/binary-com/deriv-app/pull/13254) is almost merged. The rest looks good

Reply:
Yes @sergei-deriv. Lets wait for that first